### DISCUSSION TITLE
1.23.0

### DISCUSSION THREAD

**Discussion by dani-garcia** [dani-garcia/vaultwarden#2052] [General]
⚠️  **Reminder**: If you are still using the `bitwardenrs/server*` Docker images, you need to migrate to the new `vaultwarden` image. Check https://github.com/dani-garcia/vaultwarden/discussions/1642 for an explanation. The old images are deprecated and will stop being updated after 1.23.0.

- Added emergency access feature
  - Can be disabled setting `EMERGENCY_ACCESS_ALLOWED=false` 
- Added support for single organization policy
- Fixed incorrect webauthn origin
- Enforce personal ownership policy on imports
- Fixed issue using uppercase characters on emails
- Updated web vault to 2.23.0
- Added organization bulk user management actions (reinvite/confirm/delete)
- Removed limmit that disabled sending ciphers with attachments
- Disabled enforcing of two factor organization policy on users that haven't been accepted yet
- Added tzdata to the alpine containers, to be able to set a different timezone to UTC
- Updated icon fetching to make it work on unicode websites
- Docker images are now built using Github Actions, and the base images have been updated
- Added database connection check to `/alive` endpoint
- Updated dependencies

<hr /><em>This discussion was created from the release <a href='https://github.com/dani-garcia/vaultwarden/releases/tag/1.23.0'>1.23.0</a>.</em>

**Comment by DonSYS91** [comment]
Please update the docker latest tag on the docker hub, it still points to 1.22.2

**Reply by BlackDex** [reply]
Just have a little patients. The build process wasn't done yet when you posted this. It was done just about 16min ago.

**Comment by jjlin** [comment]
> * Added emergency access feature
>   * Can be disabled setting EMERGENCY_ACCESS_ALLOWED=true

should be `false`.

**Comment by NoseyNick** [comment]
Woohoo! It's cool to go to `/admin/diagnostics` and see it all **GREEN** again. Thanks!   :grinning:

**Comment by alexschomb** [comment]
Thanks for the update!

I know this is not something supported by upstream Bitwarden (yet?), but I think that it would be helpful to support some kind of automatic emergency contact for company environments. Use case: Employee leaves the company, but didn't add (or even removed afterwards) the emergency contact of his boss (or IT department), resulting in data loss of unshared credentials. (which is common, although I agree that everybody "should" use the share function)

**Reply by BlackDex** [reply]
That is where Master Password reset is for. Which isn't implemented within Vaultwarden (yet).

**Reply by alexschomb** [reply]
Thanks for clarification. I wasn't aware of that feature! Looking forward to it being added in the future :) Thanks for your work!

In case someone else was wondering, this seems to be the feature: https://bitwarden.com/help/article/admin-reset/

**Comment by asd2003ru** [comment]
Thanks for update !
How change time zone in container?

**Reply by cksapp** [reply]
Best suited question for our support forums https://vaultwarden.discourse.group

But to answer the question this can be done with an environment variable within the container
- e TZ=America/New_York
https://serverfault.com/questions/683605/docker-container-time-timezone-will-not-reflect-changes

You can also pass local time files from the host to the container read only and this will pass local time on the machine to the container.
You will also need to make sure your logging is set to have the same time zone as well.

**Comment by asanDD** [comment]
Hi, I can't build this version. Cargo exits with this error:

Updating crates.io index
error: failed to select a version for the requirement `bytes = "^1.1.0"`
candidate versions found which didn't match: 1.0.1, 1.0.0, 0.6.0, ...
location searched: crates.io index
required by package `vaultwarden v1.0.0 (/opt/update/tmp_vaultwarden/server/vaultwarden-1.23.0)`

Cargo version: cargo 1.57.0-nightly (7fbbf4e8f 2021-10-19)
OS: Ubuntu 20.04.3 LTS

Just want to mention that and hope this is the right place.

**Reply by BlackDex** [reply]
See: https://github.com/dani-garcia/vaultwarden/discussions/2056

**Reply by asanDD** [reply]
Thank you, I didn't see that.
